spec/integration/schema_parser_spec.rb

```diff
@@ -10,7 +10,15 @@
             "attributes": {
               "name": "About naming stuff and cache invalidation"
             }
-          }
+          },
+          "included": [
```

Comment:
Why should case 2 fail?  I think it makes more sense to ignore unknown includes, considering that attributes which are not part of the schema (i.e. neither `required` or `optional`) are ignored as well.